components/brave_perf_predictor/browser/predictor.cc

```diff
@@ -0,0 +1,76 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_perf_predictor/browser/predictor.h"
+
+#include <numeric>
+#include <cmath>
+#include <utility>
+#include <algorithm>
+
+#include "base/logging.h"
+
+namespace brave_perf_predictor {
+
+bool standardise_feats_no_outliers(
```

Comment:
if this is strictly internal it should be in an anonymous namespace inside `brave_perf_predictor`